src/services/communications/Ethernet_packets.py

```diff
@@ -0,0 +1,52 @@
+import struct
+
+
+class Packets:
+    def __init__(self,*packets): 
+        #packets is a list where first goes an ID uint16_t and then string with types
+        self._dict_packets = {}
+        for packet in packets:
+            if len(packet) < 1:
+                raise ValueError ("Every packet need a ID at least")
+            if not isinstance(packet[0],int) or not (0 <= packet[0] <= 0xFFFF):
+                raise ValueError(f"The id must be an uint16_t, {packet[0]}")
+            id = packet[0]
+            self._dict_packets[id] = list()
+            for data_type in packet[1:]:
+                self._dict_packets[id].append(data_type)
```

Comment:
with a modification to the input format you just need to check that every id is an uint16